python/pyarrow/_csv.pyx

```diff
@@ -359,6 +500,11 @@ cdef class ParseOptions(_Weakrefable):
         """
         Whether two quotes in a quoted CSV value denote a single quote
         in the data.
+
+        Examples:
+        ---------
+        >>> parse_options = csv.ParseOptions(double_quote=False)
+        >>> csv.read_csv(input_file, parse_options=parse_options)
```

Comment:
I think we don't necessarily have to add an example if the example doesn't show anything interesting :-)